spec/features/navigation_bar_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+feature 'Navigation Bar' do
+  describe 'Resume options' do
+    let(:user) { create(:user) }
+
+    context 'when the user is logged in' do
+      before do
+        sign_in user
+      end
+
+      it 'has the education experience dropdown option on the navigation bar', agreggate_failures: true do
+        visit root_path
+
+        expect(page).to have_selector('.dropdown-toggle', text: t('refills.navigation.my_resume'))
+        click_link t('refills.navigation.my_resume')
```

Comment:
use the raw text instead of the translation helper, this way if there's a problem with the translation, the task will catch